### PR TITLE
Add cute robot animation

### LINKED ISSUE #380: Cute Android robot animation
## Kind (Required)

- Improvement

 ## Overview (Required)

- 
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by b4tchkn**
## Issue
- close #380

## Overview (Required)
- Add cute droid robot animation at end of  feedlist in Home.
- Is this cute? If you have better animation ideas, please let me know!

## Links
-

## Screenshot
Before | After
:--: | :--:
<video src="https://user-images.githubusercontent.com/28699251/112174302-a689b800-8c39-11eb-979b-47cbe3c6052d.mp4" width="300" /> | <video src="https://user-images.githubusercontent.com/28699251/112335133-52490b80-8cff-11eb-9295-7154ecaf2577.mp4" width="300" />

### CODE REVIEW

**Comment by takahirom** [comment]
Thank you! It feels good enough! This is my taste, but what about anime that bounces up a little? 😄

**Comment by b4tchkn** [comment]
> Thank you! It feels good enough! This is my taste, but what about anime that bounces up a little? 😄

Thank you for your idea! I fixed animation and I updated After Screenshot movie. Please confirm 🙏

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -72,15 +72,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```
Comment (takahirom) [inline .idea/codeStyles/Project.xml:75]: This change may have had a diff in my environment as well, so I'll add it! Thank you!

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FakeFeedViewModel.kt [hunk 2]
```diff
@@ -62,7 +62,7 @@ class FakeFeedViewModel(val errorFetchData: Boolean) : FeedViewModel {
             val filteredFeed = feedContents.filtered(filters)
             FeedViewModel.State(
                 filters = filters,
-                filteredFeedContents = filteredFeed
+                filteredFeedContents = filteredFeed,
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FakeFeedViewModel.kt:65]: 👍

**Review by takahirom — APPROVED** [review]
LGTM 👍

**Pull request merged**